### PR TITLE
update reward modifier params

### PR DESCRIPTION
**Pull request by Zeegomo**
Update agreement rewards modifiers for fund8

### CODE REVIEW

File: src/rewards/veterans.rs [hunk 1]
```diff
@@ -249,23 +251,15 @@ mod tests {
...
-                Rewards::new(75, 2),
-                Rewards::from(5) / Rewards::from(6),
-                Rewards::ONE,
-            ),
-            (Rewards::new(9, 1), Rewards::ONE, Rewards::ONE),
+            (Rewards::new(6, 1), Rewards::ZERO, Rewards::ZERO),
+            (Rewards::new(7, 1), Rewards::new(75, 2), Rewards::ONE),
+            (Rewards::new(8, 1), Rewards::ONE, Rewards::ONE),
+            (Rewards::new(9, 1), Rewards::new(125, 2), Rewards::ONE),
         ];
```
Comment (enzoc4) [inline src/rewards/veterans.rs:261]: Unless I'm reading these numbers wrong, I feel like you may as well just write something like:

```rust
        let agreements = [
            Rewards::new(6, 1),
            Rewards::new(7, 1),
            Rewards::new(8, 1),
            Rewards::new(9, 1),
        ];

        let inputs = agreements.iter().copied().map(|agreement| {
            (
                agreement,
                rewards_disagreement_modifier(agreement),
                reputation_disagreement_modifier(agreement),
            )
        });

```

File: src/rewards/veterans.rs [hunk 2]
```diff
@@ -40,23 +40,25 @@ fn calc_final_ranking_per_review(rankings: &[impl Borrow<VeteranRankingRow>]) ->
     }
 }
 
-fn rewards_disagreement_discount(agreement_rate: Decimal) -> Decimal {
+fn rewards_disagreement_modifier(agreement_rate: Decimal) -> Decimal {
     // Cannot use decimal range pattern in matches, and don't want to complicate
     // stuff by using exact integer arithmetic since it's not really needed at this point
-    // Thresholds at 0.9 - 0.75 - 0.6
+    // Thresholds at 0.9 - 0.8 - 0.7
```
Comment (enzoc4) [inline src/rewards/veterans.rs:46]: what about defining a `const THRESHOLDS: [Decimal; 3] = [...]` somewhere?

**Review by enzoc4 — COMMENTED** [review]
I'm commenting on a few unrelated things (I think) that caught my attention, I'm actually not sure where are the specs that these changes follow, so I'm not sure if I should approve it. If I manage to find the specs for this I'll take a closer look at the numbers, although I believe this is probably good.

File: src/rewards/veterans.rs (hunk 1 above)
Comment (Zeegomo) [reply src/rewards/veterans.rs:261]: Yes, it was a way to also check that `rewards_disagreement_modifier` and `reputation_disagreement_modifier` work as expected

File: src/rewards/veterans.rs [hunk 3]
```diff
@@ -40,34 +47,25 @@ fn calc_final_ranking_per_review(rankings: &[impl Borrow<VeteranRankingRow>]) ->
...
-    if agreement_rate >= Decimal::new(6, 1) {
-        Decimal::ONE
-    } else {
-        Decimal::ZERO
+fn disagreement_modifier(
+    agreement_rate: Decimal,
+    thresholds: impl IntoIterator<Item = Decimal>,
+    modifiers: impl IntoIterator<Item = Decimal>,
+) -> Decimal {
+    for (threshold, modifier) in thresholds.into_iter().zip(modifiers) {
```
Comment (enzoc4) [inline src/rewards/veterans.rs:55]: neat, it certainly looks better.

**Review by enzoc4 — APPROVED** [review]

**Pull request merged**